packages/x/xterm.json

```diff
@@ -1,37 +1,34 @@
 {
   "name": "xterm",
-  "filename": "xterm.min.js",
-  "description": "Full xterm terminal, in your browser",
+  "filename": "xterm.js",
+  "description": "Build terminals in the browser",
   "homepage": "https://xtermjs.org/",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/sourcelair/xterm.js"
+    "url": "git+https://github.com/xtermjs/xterm.js"
   },
-  "keywords": [
-    "xterm",
-    "terminal",
-    "browser-tty"
-  ],
+  "keywords": ["xterm", "terminal", "browser-tty"],
   "license": "MIT",
   "autoupdate": {
     "source": "npm",
-    "target": "xterm",
+    "target": "@xterm/xterm",
     "fileMap": [
       {
-        "basePath": "dist",
-        "files": [
-          "**/*"
-        ]
+        "basePath": "lib",
+        "files": ["*.js", "*.map"]
+      },
+      {
+        "basePath": "css",
+        "files": ["*.css"]
       }
-    ],
-    "ignoreVersions": [
-      "3.*-beta*",
-      "3.14.0"
     ]
   },
+  "optimization": {
+    "js": false
+  },
```

Comment:
Is there a reason for disabling JS optimization as part of this PR? Was there an issue with the minification in previous versions?

Reply:
The source JS is already minified. I guess the source map doesn't make much sense if we further minify the code?

Reply:
This seems to prevent you to merge the PR, right? Anything I can do to help?

Reply:
Disabling it is fine, just wanted to check it was intentional :)